apis/config/v1beta1/configuration_types.go

```diff
@@ -588,3 +592,31 @@ type WorkloadRetentionPolicy struct {
 	// +optional
 	AfterDeactivatedByKueue *metav1.Duration `json:"afterDeactivatedByKueue,omitempty"`
 }
+
+type FailureRecoveryPolicy struct {
```

Comment:
Let's take a backup of the work, but adapt the PR to align with the most recent KEP